apps/web/src/components/display/ApplicantTable.tsx

```diff
@@ -17,25 +18,36 @@ export const isHired = (id?: number) => {
   return id === STATUS.Candidate_accepted_the_job_offer;
 };
 
-const milestoneText = (id?: number, status?: string) => {
+// determine milestone text for status
+const milestoneText = (id?: number, status?: IENApplicantStatusRO, haId?: number | null) => {
   if (!id && !status) {
     return <td className='px-6'></td>;
   }
 
-  return isHired(id) ? (
-    <td className='px-6 font-bold text-bcGreenHiredText'>
-      Hired
-      <img src={hiredCheckmarkIcon.src} alt='hired checkmark' className='inline-block h-6 mx-2' />
-    </td>
+  // if applicant has accepted an offer, show detailed Hired status and checkmark
+  if (isHired(id)) {
+    return (
+      <td className='px-6 font-bold text-bcGreenHiredText'>
+        Hired
+        <img src={hiredCheckmarkIcon.src} alt='hired checkmark' className='inline-block h-6 mx-2' />
+      </td>
+    );
+  }
+
+  // else check if current user is part of a HA and the status is in the Recruitment step
+  return haId && status?.parent?.id === STATUS.IEN_Recruitment ? (
```

Comment:
users from hmbc, moh, phsa also can't see the recruitment status. Is it ok?

Reply:
hmbc and moh users don't actually get assigned a HA id, as of writing this at least.  So they can see the recruitment status instead of 'In Progress'